router/http_test.go

```diff
@@ -1005,6 +1004,7 @@ func (s *S) TestDefaultServerKeypair(c *C) {
 	assertGet(c, "https://"+l.TLSAddr, "foo.example.com", "2")
 }
 
+// TODO(bgentry): validate this in Postgres??
 func (s *S) TestCaseInsensitiveDomain(c *C) {
```

Comment:
thoughts on this one? It's great that the router can work with non-lowercase domains, but maybe we should enforce that before shoving the data into Postgres.